packages/store/types/Drizzle.d.ts

```diff
@@ -0,0 +1,33 @@
+import { Store } from 'redux';
+import { IStoreConfig } from './generateStore';
+import { IContract } from './IContract';
+import { IContractConfig } from './contractStateUtils';
+
+export interface IDrizzleOptions {
+  contracts: IContract[];
+  events?: {
+    [contractName: string]: any;
+  };
+  polls?: any;
+  syncAlways?: any;
+  web3?: {
+    customProvider?: any;
+    fallback?: {
+      type: string;
+      url: string;
+    }
+  },
+  networkWhitelist?: any[];
```

Comment:
Looks like this should be `number[]`

- https://web3js.readthedocs.io/en/v1.2.1/web3-eth-net.html#getid
- https://github.com/trufflesuite/drizzle/blob/0ee523028e72c80ca59477a86be11a58388729ae/packages/store/src/web3/web3Saga.js#L78
- https://github.com/trufflesuite/drizzle/blob/0ee523028e72c80ca59477a86be11a58388729ae/packages/store/src/drizzleStatus/drizzleStatusSaga.js#L28